bridge_adaptivity/module/views.py

```diff
@@ -61,6 +61,9 @@ def get_form_kwargs(self):
     def get_form(self, form_class=None):
         form = super().get_form()
         form.fields['owner'].initial = self.request.user.id
+        form.fields['name'].help_text = (
+            "Name of new Collection. You can choose the available collections under the Collection settings block"
```

Comment:
typo: s/Name of new Collections/Name of **the** new Collection/

s/You can choose the available/You can choose available/

Reply:
Done